src/Selenium2Library/__init__.py

```diff
@@ -78,6 +78,10 @@ class Selenium2Library(
     This can be fixed by changing the locator to:
     | Click Link    default=page?a=b
 
+    Please note that JavaScript is not provided by Selenium2Library and
+    if there is need to use JavaScript must be provided by the system
+    under test.
+
```

Comment:
Not JavaScript but jQuery. Slightly rewording it might say

> Please note that jQuery is not provided by Selenium2Library and if there is need to use jQuery locators, the system under test must provide the jQuery library.

or

> In regards to the jQuery locator stratergy, please note that jQuery is not provided by Selenium2Library. If the test uses jQuery locators, the system under test must provide the jQuery library.